app/services/container_service_mixin.rb

```diff
@@ -63,16 +64,16 @@ def utilization_data(used_cpu, total_cpu, used_mem, total_mem)
     if used_cpu.any?
       {
         :cpu => {
-          :used  => used_cpu.values.last.round,
-          :total => total_cpu.values.last.round,
+          :used  => (used_cpu.values.last || 0).round(@custom_display_precision || DISPLAY_PRECISION),
+          :total => (total_cpu.values.last || 0).round,
```

Comment:
Shouldn't this use the same precision?

(Otherwise this is the only place where `used` is calculated with different precision than `total`.)

Reply:
Total cpu should always be a round number. There usually isnt a total of `1.444455` cpu cores for a node\cluster(and i dont think we want to show it if it does happen  :sweat: ).

Reply:
Ah, got it.. so `total` is always precision=0.

We can still fit that in :)